addon/globalPlugins/EnhancedFindDialog/guiHelper.py

```diff
@@ -90,29 +90,32 @@ def __init__(self, parent, cursorManager, profile, searchEntries):
 
 		mainSizer = wx.BoxSizer(wx.VERTICAL)
 
-		findSizer = wx.BoxSizer(wx.HORIZONTAL)
+		sHelper = guiHelper.BoxSizerHelper(self, orientation=wx.VERTICAL)
 		# Translators: Dialog text for NvDA's find command.
 		textToFind = wx.StaticText(self, wx.ID_ANY, label=__("Type the text you wish to find"))
-		findSizer.Add(textToFind)
+		sHelper.addItem(textToFind)
```

Comment:
The visual layout is correct, thanks.

For now you have the search combo-box below its label; in original NVDA's find dialog however, the edit field is located at the right of its label.

Below is a 2-part suggestion to accomplish it:

First part of suggestion: remove this line.

```suggestion
```